UI/Source/AppKitExtensions/NestableScrollView.swift

```diff
@@ -0,0 +1,39 @@
+import AppKit
+
+open class NestableScrollView : NSScrollView {
```

Comment:
Totally in favor of pulling this out but seems like we need a change in CollectionViewController to allow for customizing the scrollView used in order for this to be useful within pilot right?

Can just be `makeScrollView` function in the style of `makeRootView`.

Reply:
Yeah good call, the FullWidthScrollView is now a subclass of this (so we're using it by default), but I agree we should add a `makeScrollView` function too.

Reply:
Ah. I had missed that it was a subclass. 👍